neuralpredictors/layers/shifters/base.py

```diff
@@ -0,0 +1,8 @@
+class Shifter:
+    def __repr__(self):
+        s = super().__repr__()
+        s += " [{} regularizers: ".format(self.__class__.__name__)
+        ret = []
+        for attr in filter(lambda x: "gamma" in x, dir(self)):
+            ret.append("{} = {}".format(attr, getattr(self, attr)))
+        return s + "|".join(ret) + "]\n"
```

Comment:
Would it make sense to make `Shifter` an abstract subclass of `nn.Module`? 
with additional `reuglarizer` and `initialize` methods that need to be implemented?

Reply:
yes, sounds good. will do